docs/man_pages/publishing/appmanager-upload-android.md

```diff
@@ -13,9 +13,14 @@ WARNING: This command is not applicable to mobile website projects. To view the
 ### Options
 * `--certificate` - Sets the certificate that you want to use for code signing your Android app. You can set a certificate by index or name. <% if(isHtml) { %>To list available certificates, run `$ appbuilder certificate`<% } %> 
 * `--download` - If set, downloads the application package to the root of the project.
+* `--publish` - If set, the application package will be automatically published and visible for the members of its distribution groups. To add distribution groups, use `--groups` option.
+* `--send-email` - If set, an email with the link to the app will be send to all eligible users.
```

Comment:
Totally agree! I like your messages too
@ikoevska could you please confirm about these warnings?